lib/hexpm/repository/package_download.ex

```diff
@@ -37,6 +37,16 @@ defmodule Hexpm.Repository.PackageDownload do
     )
   end
 
+  def packages_and_all_download_views(packages) do
+    package_ids = Enum.map(packages, &(&1.id))
+
+    from(pd in PackageDownload,
+      join: p in assoc(pd, :package),
+      where: pd.package_id in ^package_ids,
+      select: {p.id, pd.view, coalesce(pd.downloads, 0)},
+      order_by: [desc: pd.view])
```

Comment:
`order_by` is not needed since we will put it in a map later, right?

Reply:
Removed order_by.